pkg/sdk/testint/procedures_integration_test.go

```diff
@@ -0,0 +1,536 @@
+		require.Equal(t, nid.Name(), e.Name)
+	})
+
+	t.Run("alter procedure: set log level", func(t *testing.T) {
+		f := createProcedureForSQLHandle(t, true)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, f.Name)
+		set := sdk.NewProcedureSetRequest().WithLogLevel(sdk.String("DEBUG"))
+		request := sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithSet(set)
+		err := client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+	})
+
+	t.Run("alter procedure: set trace level", func(t *testing.T) {
+		f := createProcedureForSQLHandle(t, true)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, f.Name)
+		set := sdk.NewProcedureSetRequest().WithTraceLevel(sdk.String("ALWAYS"))
+		request := sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithSet(set)
+		err := client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+	})
+
+	t.Run("alter procedure: set comment", func(t *testing.T) {
+		f := createProcedureForSQLHandle(t, true)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, f.Name)
+		set := sdk.NewProcedureSetRequest().WithComment(sdk.String("comment"))
+		request := sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithSet(set)
+		err := client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+	})
+
+	t.Run("alter procedure: unset comment", func(t *testing.T) {
+		f := createProcedureForSQLHandle(t, true)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, f.Name)
+		unset := sdk.NewProcedureUnsetRequest().WithComment(sdk.Bool(true))
+		request := sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithUnset(unset)
+		err := client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+	})
+
+	t.Run("alter procedure: set execute as", func(t *testing.T) {
+		f := createProcedureForSQLHandle(t, true)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, f.Name)
+		request := sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithExecuteAs(sdk.ProcedureExecuteAsOwner)
+		err := client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+	})
+
+	t.Run("alter procedure: set and unset tags", func(t *testing.T) {
+		f := createProcedureForSQLHandle(t, true)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, f.Name)
+		setTags := []sdk.TagAssociation{
+			{
+				Name:  tagTest.ID(),
+				Value: "abc",
+			},
+		}
+		request := sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithSetTags(setTags)
+		err := client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+
+		unsetTags := []sdk.ObjectIdentifier{
+			tagTest.ID(),
+		}
+		request = sdk.NewAlterProcedureRequest(id).WithArgumentTypes(defaultArgumentTypes).WithUnsetTags(unsetTags)
+		err = client.Procedures.Alter(ctx, request)
+		require.NoError(t, err)
+	})
+
```

Comment:
There's no "show" test that would assert on every field content and "desc" where you would take few "property"-ies and assert on their "value"s